it_zwierzak/adoption_announcement/migrations/0002_animal_announcment_remove_buttonquestion_button_and_more.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 4.0.6 on 2023-01-22 10:13
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('adoption_announcement', '0001_initial'),
+    ]
```

Comment:
Generalnie jest taka zasada, że jeden pull request powinien mieć jedną migrację, czyli jeden zestaw zmian w bazie danych, czasem może być problem z migracją danych i trzeba to rozbić na więcej, ale jak się dodaje tak dużo tych migracji tzn, że ktoś po prostu dużo pomysłów miał i dużo zmian było